components/dashboard/src/components/DropDown2.tsx

```diff
@@ -136,7 +136,7 @@ export const DropDown2: FunctionComponent<DropDown2Props> = (props) => {
                 className={
                     "h-16 bg-gray-100 dark:bg-gray-800 flex items-center px-2 " +
                     (showDropDown
-                        ? "rounded-t-lg"
+                        ? "rounded-t-lg drop-shadow-xl"
```

Comment:
do we also need the `filter` class here like we did for context menu because of our version of tailwind?